src/main/java/hudson/plugins/gradle/injection/maven/GradleEnterpriseExtensionsContributorFactory.java

```diff
@@ -0,0 +1,94 @@
+package hudson.plugins.gradle.injection.maven;
+
+import com.google.common.base.Splitter;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.maven.PlexusModuleContributor;
+import hudson.maven.PlexusModuleContributorFactory;
+import hudson.model.AbstractBuild;
+import hudson.model.Computer;
+import hudson.model.Node;
+import hudson.plugins.gradle.injection.MavenBuildScanInjection;
+import hudson.util.LogTaskListener;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+@Extension(optional = true)
+public class GradleEnterpriseExtensionsContributorFactory extends PlexusModuleContributorFactory {
+
+    private static final Logger LOGGER = Logger.getLogger(GradleEnterpriseExtensionsContributorFactory.class.getName());
+
+    private static final PlexusModuleContributor EMPTY_CONTRIBUTOR = PlexusModuleContributor.of();
+    private static final Splitter UNIX_CLASSPATH_SPLITTER = Splitter.on(':').omitEmptyStrings();
```

Comment:
I think this can be replaced by `File.pathSeparator`

Reply:
I was also thinking about it. Not 100% sure. This particular code is consuming what was produced by this code https://github.com/c00ler/gradle-plugin/blob/master/src/main/java/hudson/plugins/gradle/injection/MavenBuildScanInjection.java#L138. It gathers all the information from the computer. Since I'm not 100% sure that it can be replaced with just `File.pathSeparator` I decided to leave it as is. And implemented a similar check on this side.

Reply:
I think both can be replaced. The documentation says:

> The system-dependent path-separator character. This field is initialized to contain the first character of the value of the system property path.separator. This character is used to separate filenames in a sequence of files given as a path list. On UNIX systems, this character is ':'; on Microsoft Windows systems it is ';'.

Reply:
But let's do this in the follow-up release.